backend/controller/getAuthRoles.js

```diff
@@ -5,28 +5,35 @@ const {
     DIRECTOR_TITLES,
 } = require('../models/constants')
 
-const getAuthRoles = async (identifier) => {
-    const currentGoogleGroup = await getGoogleGroup(identifier)
+const getWATIAMFromEmail = async (email) => {
+    return email.substring(0, email.indexOf('@'))
+}
+
+const getAuthRoles = async (user_uid, email, reporter_id) => {
+    const watiam = await getWATIAMFromEmail(email)
+    const currentGoogleGroup = await getGoogleGroup(watiam)
 
-    let isAdmin, isTeamCaptain, isDirector
+    let isAdmin, isTeamCaptain, isDirector, isReporter
 
     if (process.env?.AUTH_OVERRIDE) {
-        _isAdmin = process.env.AUTH_OVERRIDE === 'ADMIN'
-        _isTeamCaptain =
-            process.env.AUTH_OVERRIDE === 'TEAM_CAPTAIN' || _isAdmin
-        _isDirector = process.env.AUTH_OVERRIDE === 'DIRECTOR' || _isTeamCaptain
+        isAdmin = process.env.AUTH_OVERRIDE === 'ADMIN'
+        isTeamCaptain = process.env.AUTH_OVERRIDE === 'TEAM_CAPTAIN' || isAdmin
+        isDirector = process.env.AUTH_OVERRIDE === 'DIRECTOR' || isTeamCaptain
+        isReporter = process.env.AUTH_OVERRIDE === 'REPORTER'
```

Comment:
we should assume `AUTH_OVERRIDE` is a comma separated list and check if the permission level is in the list. ex. `authOverrides = process.env.AUTH_OVERRIDE.split(',')` and  `authOverrides.contains('REPORTER')`